<launch>
  <arg name = "startX" value="10" /> <!--  X coordinate of the starting point -->
  <arg name = "startY" value="10" /> <!--  Y coordinate of the starting point -->
  <arg name = "endX" value="480" /> <!--  X coordinate of the goal -->
  <arg name = "endY" value="480" /> <!--  Y coordinate of the goal -->
  <arg name = "stepSize" value="12.0" /> <!--step size of RRT -->
  <arg name = "maxIter" value="20000" /><!--max iterations of the RRT algo-->
  <arg name = "animate" value="true" /> <!--set to true to visualize RRT-->
  <arg name="map_file" value="/home/raakesh/Documents/cppprojects/catkin_ws/src/rrt/maps/map.yaml"/> 
  <node pkg="map_server" type="map_server" name="map_server" args="$(arg map_file)"/>
  <node pkg="rrt" type="rrt_node" name="planner" args="_startX:=$(arg startX) _startY:=$(arg startY) _endX:=$(arg endX) _endY:=$(arg endY) _stepSize:=$(arg stepSize) _maxIter:=$(arg maxIter) _animate:=$(arg animate)" output="screen"/>
</launch>